### PR TITLE
app-admin/cdist: bump to 5.1.3

### PR DESCRIPTION
**Pull request by dheule**
Package-Manager: Portage-2.3.69, Repoman-2.3.16
Signed-off-by: Daniel Heule <nerve@ipwar.ch>

### CODE REVIEW

File: app-admin/cdist/cdist-5.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+PYTHON_COMPAT=( python{3_5,3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A usable configuration management system"
+HOMEPAGE="https://www.nico.schottelius.org/software/cdist/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (juippis) [inline app-admin/cdist/cdist-5.1.3.ebuild]: `GNU General Public License v3 or later (GPLv3+)`

File: app-admin/cdist/cdist-5.1.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_5,3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A usable configuration management system"
+HOMEPAGE="https://www.nico.schottelius.org/software/cdist/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```
Comment (juippis) [inline app-admin/cdist/cdist-5.1.3.ebuild]: I see upstream has some sort of tests included at least in their code repo, 
https://code.ungleich.ch/ungleich-public/cdist/tree/master/cdist/test
if they are available in the pypi tarball could you look into implementing them in the ebuild?

File: app-admin/cdist/cdist-5.1.3.ebuild (hunk 1 above)
Comment (dheule) [reply app-admin/cdist/cdist-5.1.3.ebuild]: Sorry, overseen, now fixed with lastest commit

File: app-admin/cdist/cdist-5.1.3.ebuild (hunk 2 above)
Comment (dheule) [reply app-admin/cdist/cdist-5.1.3.ebuild]: This test are unittest to test the python code (cdist core) after changes, we run the tests bevor we release to pypi. In consequence, the test is NOT included in the release tarbal. We have only python as dependency for the core, so there is also no benefit to run the tests again on each and every system.

Comment (juippis) [reply app-admin/cdist/cdist-5.1.3.ebuild]: Right, thanks!

**Pull request closed without merging**